ca.hamaluik.Timecop.yml

```diff
@@ -0,0 +1,33 @@
+app-id: ca.hamaluik.Timecop
```

Comment:
Given that the app website is `timecop.app`, I was wondering if that was the best choice.

Reply:
I built the app before I registered the domain, unfortunately..